hey-car/heycar-uikit
# HEYUI-60 Implementation Font package

## Pull Request

### Description
Adds heycar fonts as css fonts face
Adds heycar fonts as css in js

Describe your changes (or add ticket description) and add relevant screenshots

### Type of change


### How do I test this

run `npm run build`
go to dist folder inside fonts package run `npm link`,
in es-web-app run  `npm link @heycar-uikit/fonts`
in _app.js import like this `import '@heycar-uikit/fonts'` 
run `npm run dev` and see the fonts loaded in the browser

## Checklist

### Did you remember to take care of the following?

- [x] `npm i` – for new NPM dependencies.
- [ ] `npm run lint` - to check for linting issues
- [ ] `npm run test` - to run unit tests
- [ ] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix


Thanks for contributing!
